### PR TITLE
Sort the list of workspace package versions

### PR DESCRIPTION
**Pull request by jtpio**
At the moment the changelog shows a list of unordered npm packages:

For example: https://github.com/jupyterlab/jupyterlab/pull/11651

This change sorts them so it's a bit more pleasing to the eye.

### CODE REVIEW

File: jupyter_releaser/npm.py [hunk 1]
```diff
@@ -149,7 +149,7 @@ def get_package_versions(version):
         message += f'\nnpm version: {data["name"]}: {npm_version}'
     if "workspaces" in data:
         message += "\nnpm workspace versions:"
-        for path in _get_workspace_packages(data):
+        for path in sorted(_get_workspace_packages(data)):
```
Comment (blink1073) [inline jupyter_releaser/npm.py:152]: Maybe we should sort the result returned by that function so it is always sorted?

Comment (blink1073) [reply jupyter_releaser/npm.py:152]: (in the function itself)

Comment (jtpio) [reply jupyter_releaser/npm.py:152]: Yes sounds like it would be more convenient.

**Review by blink1073 — APPROVED** [review]
Thanks!

**Pull request merged**